### ISSUE TITLE
Insistence on using index.html as the pagination template

### ISSUE THREAD

**Issue by eyalroz** [jekyll/jekyll-paginate#37]
I'm using the [Beautiful Jekyll](https://github.com/daattali/beautiful-jekyll) theme to create a small personal website. On my site, I don't want posts to appear in the front page, the index page; so I removed the relevant code from `index.html` (in fact, I got rid of it in favor of an `index.md` with static content). Now I get the warning message:

```
Pagination: Pagination is enabled, but I couldn't find an index.html page to use as the pagination template. Skipping pagination.
```
I found [this issue page](https://github.com/jekyll/jekyll/issues/3282) on this matter at the [Jekyll core project](https://github.com/jekyll/jekyll/), where it's claimed this is a Jekyll-paginate issue. Well, here I am reporting it to you :-(

I'll state that I'm willing to move the pagination code elsewhere, so maybe there's some workaround in my case.

**Comment by ishwon** [comment]
I'm using the following:
  * jekyll (3.4.3)
  * jekyll-paginate (1.1.0)

I'm encountering the same problem as eyalroz.

**Comment by michaellee** [comment]
I found this to be helpful https://stackoverflow.com/a/29177455/703220

Basically do as the warning tells you, instead of using a non-index.html file use an index.html.

In my case, I wanted to put pagination on a file called archive.html.

The solution was to put created a folder called archive and place a file called index.html

Then just update the permalink and `paginate_path`

**Comment by eyalroz** [comment]
@michaellee : So, you're essentially suggesting the use of an alternate version of jekyll-paginate? I might look into it, but that does not resolve the issue for this package (= jekyll/jekyll-paginate).

**Comment by michaellee** [comment]
Seeing as this plugin isn't under active development, thought it was a valid solution to point to a project that fixed my problem is all.

**Comment by naveenjujaray** [comment]
yeah i agree with that paginate-v2 is not compatible with github pages
